rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/attachment/Sw360AttachmentService.java

```diff
@@ -569,4 +575,80 @@ public void fillCheckedAttachmentData(Attachment attachment, User user) throws S
             attachment.unsetCheckedOn();
         }
     }
+
+    public boolean isValidSbomFile(MultipartFile file) throws TException {
+
+        if (file == null || file.isEmpty()) {
+            return false;
+        }
+
+        String originalFilename = file.getOriginalFilename();
+        File tmpFile = null;
+
+        try {
+            String lowerCaseFilename = originalFilename.toLowerCase();
+
+            // SPDX formats
+            if (lowerCaseFilename.endsWith(".spdx") || lowerCaseFilename.endsWith(".rdf")) {
+
+                String extension = lowerCaseFilename.endsWith(".spdx") ? ".spdx" : ".rdf";
+                tmpFile = saveAsTempFile(file.getInputStream(), "sbom_validation", extension);
+
+                try {
+                    SpdxDocument doc = deserializeDocument(tmpFile);
```

Comment:
This has been update din new version
```suggestion
                    SpdxDocument doc = SpdxToolsHelper.deserializeDocumentCompatV2(tmpFile);
```

Reply:
DONE in https://github.com/eclipse-sw360/sw360/pull/3148